controllers/suite_test.go

```diff
@@ -78,3 +85,60 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+var _ = Describe("Reconcile Test", func() {
+	When("an Operator is created", func() {
+		var (
+			operator *operatorsv1alpha1.Operator
+			ctx      context.Context
+			opName   string
+			pkgName  string
+			err      error
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+
+			opName = fmt.Sprintf("operator-test-%s", rand.String(8))
+			pkgName = fmt.Sprintf("package-test-%s", rand.String(8))
+
+			operator = &operatorsv1alpha1.Operator{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: opName,
+				},
+				Spec: operatorsv1alpha1.OperatorSpec{
+					PackageName: pkgName,
+				},
+			}
+			err = k8sClient.Create(ctx, operator)
+			Expect(err).To(Not(HaveOccurred()))
+
+			or := controllers.OperatorReconciler{
+				k8sClient,
+				scheme.Scheme,
+			}
+			_, err = or.Reconcile(ctx, reconcile.Request{
+				NamespacedName: types.NamespacedName{
+					Name: opName,
+				},
+			})
+			Expect(err).To(Not(HaveOccurred()))
+		})
+		AfterEach(func() {
+			err = k8sClient.Delete(ctx, operator)
+			Expect(err).To(Not(HaveOccurred()))
+		})
+		It("has a Condition created", func() {
+			getOperator := &operatorsv1alpha1.Operator{}
+
+			err = k8sClient.Get(ctx, client.ObjectKey{
+				Name: opName,
+			}, getOperator)
+			Expect(err).To(Not(HaveOccurred()))
+
+			// There should always be a "Ready" condition, regardless of Status.
+			conds := getOperator.Status.Conditions
+			Expect(conds).To(Not(BeEmpty()))
+			Expect(conds).To(ContainElement(HaveField("Type", operatorsv1alpha1.TypeReady)))
```

Comment:
Should we also test for the specific Status and Reason we expect?

Reply:
I decided to not bother with that. As I wanted to minimize changes (vs. additions) to the tests as code is added.